util/greeting-sentences.ts

```diff
@@ -0,0 +1,8 @@
+const greetingSentences = (
```

Comment:
Where would these scripts be used?

Reply:
I just want to create these function for others can be used to make a landing page or something else Lol.

Reply:
I don't think it is necessary in this point of time krub.